package.json

```diff
@@ -90,7 +90,6 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^2.0.5",
     "qunit": "^2.11.0",
-    "qunit-dom": "^1.4.0",
```

Comment:
qunit-dom is not compatible (yet, working on a PR) with the latest ember-qunit beta, and since we don't directly use it I figured I'd just remove it...

Reply:
Makes sense.